packages/vm/benchmarks/run.ts

```diff
@@ -1,52 +1,58 @@
 import Benchmark from 'benchmark'
 import { BenchmarksType } from './util'
 import { mainnetBlocks } from './mainnetBlocks'
+import { bytecode } from './bytecode'
+import { ArgumentParser } from 'argparse'
```

Comment:
I think I'd rather use `yargs` here since we already have it in other places in the monorepo rather than add yet another args parsing package to our overall monorepo.  I'd be happy to pick this up if you're not familiar with the API.

Reply:
Done